File: app/src/main/java/com/akay/fitnass/data/storage/model/Runs.java
package com.akay.fitnass.data.storage.model;

import android.arch.persistence.room.Entity;
import android.arch.persistence.room.PrimaryKey;

import org.threeten.bp.ZonedDateTime;

import java.util.List;

@Entity
public class Runs {

    @PrimaryKey(autoGenerate = true)
    private long id;
    private List<Lap> laps;
    private String comment;
    private ZonedDateTime dateTime;

    public long getId() {
        return id;
    }

    public void setId(long id) {
        this.id = id;
    }

    public List<Lap> getLaps() {
        return laps;
    }

    public void setLaps(List<Lap> laps) {
        this.laps = laps;
    }

    public String getComment() {
        return comment;
    }

    public void setComment(String comment) {
        this.comment = comment;
    }

    public ZonedDateTime getDateTime() {
        return dateTime;
    }

    public void setDateTime(ZonedDateTime dateTime) {
        this.dateTime = dateTime;
    }
}

File: app/src/main/java/com/akay/fitnass/data/model/Runs.kt
package com.akay.fitnass.data.model

import androidx.room.Entity
import androidx.room.PrimaryKey
import org.threeten.bp.ZonedDateTime

@Entity
data class Runs(
        @PrimaryKey(autoGenerate = true)
        var id: Long? = null,
        var laps: List<Lap>,
        var comment: String? = null,
        var dateTime: ZonedDateTime
)
